core/src/main/scala/kafka/log/LocalLog.scala

```diff
@@ -1005,12 +1005,12 @@ object LocalLog extends Logging {
 
   private[log] def emptyFetchDataInfo(fetchOffsetMetadata: LogOffsetMetadata,
                                       includeAbortedTxns: Boolean): FetchDataInfo = {
-    val abortedTransactions =
-      if (includeAbortedTxns) Some(List.empty[FetchResponseData.AbortedTransaction])
-      else None
-    FetchDataInfo(fetchOffsetMetadata,
+    val abortedTransactions: Optional[java.util.List[FetchResponseData.AbortedTransaction]] =
+      if (includeAbortedTxns) Optional.of(List.empty[FetchResponseData.AbortedTransaction].asJava)
```

Comment:
We can pass `Collections.emptyList` instead of creating a Scala list and then converting to Java.